Source/WarpX.cpp

```diff
@@ -3666,15 +3410,15 @@ WarpX::getFieldDotMaskPointer ( FieldType field_type, int lev, int dir ) const
 }
 
 void WarpX::SetDotMask( std::unique_ptr<amrex::iMultiFab>& field_dotMask,
-                        FieldType field_type, int lev, int dir ) const
+                        std::string field_name, int lev, int dir ) const
```

Comment:
Clang-tidy error "the parameter 'field_name' is copied for each invocation but only used as a const reference; consider making it a const reference [performance-unnecessary-value-param,-warnings-as-errors]":
```suggestion
                        std::string const & field_name, int lev, int dir ) const
```

Reply:
Fixed manually in [4f499d0](https://github.com/ECP-WarpX/WarpX/pull/5230/commits/4f499d00e29b139c85ac40b332e47b4e0c06d1c8) (needed a change in Source/WarpX.H, too).